### ISSUE TITLE
WCAG 2.0/2.1 4.1.1 Parsing

### ISSUE THREAD

**Issue by mbgower** [IBMa/equal-access#1659]
Hill:
Product teams testing for conformance with WCAG 2.0 and 2.1 will never encounter violations, reviews or recommendations for 4.1.1

Playback Demonstration:
1. A team testing with the WCAG 2.0 or 2.1 rulesets will never encounter any issues mapped to 4.1.1, even when content exists which would have previously triggered errors for 4.1.1
2. A team testing with the 2.0/2.1 rulesets will still encounter errors where parsing problems lead to an accessibility issue; however, they will report to a different criterion than 4.1.1
3. A team generating a new ACR for WCAG 2.0/2.1 will always report 4.1.1 as being met, even if the product's existing ACR shows it as not being met.

Metric: 
The __ (# of) rules mapping to 4.1.1, either map to a different appropriate criterion, or are removed.

[put into future revamp]
Is this practical? A query against the ACS system shows a baseline that __ current ACRS report something other than "Pass" for 4.1.1. A concluding query shows that all current ACRs report 4.1.1 as being met.

Comms:
1. Communication will come out as part of the regular release cadence for checker (a note in the What's New section).
2. The Rules Help file for 4.1.1 will be updated
3. As well, the change will be covered in an accessibility update call.
4. 5.4 release communications
